wagtail/wagtailadmin/static_src/wagtailadmin/scss/layouts/404.scss

```diff
@@ -0,0 +1,101 @@
+@import 'wagtailadmin/scss/variables';
+@import 'wagtailadmin/scss/mixins';
+@import 'wagtailadmin/scss/grid';
+@import 'wagtailadmin/scss/components/logo';
+
+body {
+    font-family: $font-sans;
+    color: $color-white;
+
+    .page404__bg {
+        position: fixed;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+        background-color: $color-teal-darker;
+
+        .wagtail-logo-container__desktop {
+            position: fixed;
+            right: calc(50% + 130px);
+            top: calc(45% - 250px);
+            width: 400px;
+            height: 500px;
+
+            .wagtail-logo {
+                width: 100%;
+                height: 100%;
+            }
+        }
+
+        .page404__text-container {
+            position: fixed;
+            left: calc(50% - 130px);
+            top: calc(45% - 150px);
+            width: 600px;
+            text-align: center;
+            z-index: 50;
```

Comment:
None of this should be necessary. Just position a container with the logo and the right-hand text, and set them to display next to one another.